mkcli.py

```diff
@@ -192,7 +206,7 @@ def run(args):
         try:
           if filesData != {}:
             requests.post(muuktestRoute + 'upload_cloud_steps_images/', headers=hed, files = filesData)
-            #requests.post(muuktestRoute + 'upload_cloud_steps_images/', data={'cloudKey': cloudKey}, headers=hed, files = filesData,  verify=False)
+            # requests.post(muuktestRoute + 'upload_cloud_steps_images/', headers=hed, files = filesData,  verify=False)
```

Comment:
leave the line as it was (including cloudkey data ).... 
             #requests.post(muuktestRoute + 'upload_cloud_steps_images/', data={'cloudKey': cloudKey}, headers=hed, files = filesData,  verify=False)